tests/ballerina-examples-test/build.gradle

```diff
@@ -0,0 +1,102 @@
+apply from: "$rootDir/gradle/javaProject.gradle"
+
+
+description = 'Ballerina - Examples Test'
+
+
+import org.apache.tools.ant.taskdefs.condition.Os
+import groovy.json.JsonSlurper
+
+
+configurations {
+    ballerinaTools {
+        transitive false
+    }
+}
+
+dependencies {
+    ballerinaTools project(path: ':ballerina-tools', configuration: 'zip')
```

Comment:
I think we also need to include implementation dependency on :ballerina-tools to make sure that the zip get's created before running the examples tests